lib/menu_types.rb

```diff
@@ -0,0 +1,8 @@
+module MenuTypes
+
+  attr_reader :CHINESE, :BRITISH
+
+  CHINESE = [{food: "Salt and Pepper King Prawn", price: 3.99, quantity_ordered: 0}, {food: "Pork Dumplings", price: 2.50, quantity_ordered: 0}]
+  BRITISH = [{food: "Fish and Chips", price: 4.50, quantity_ordered: 0}, {food: "Battered Sausage", price: 2.29, quantity_ordered: 0}]
```

Comment:
Metrics/LineLength: Line is too long. [134/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)